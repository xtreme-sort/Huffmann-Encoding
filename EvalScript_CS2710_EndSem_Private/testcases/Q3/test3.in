GenerateTreeEnhanced
52
z 1
r 3
1 4
M 4
s 4
P 5
6 7
C 7
Y 7
S 8
8 12
m 12
E 13
b 13
p 13
7 14
O 14
9 16
d 19
l 20
J 22
c 22
N 23
j 23
h 24
q 25
y 26
G 27
U 27
v 27
R 29
X 29
k 29
H 30
K 30
o 30
Q 31
u 31
a 32
F 33
I 33
x 33
3 34
Z 37
e 37
4 43
W 43
f 43
0 47
i 47
T 49
_ 49
Quit
