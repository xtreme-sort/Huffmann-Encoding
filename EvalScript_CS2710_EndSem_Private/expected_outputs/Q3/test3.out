((((((O,(C,Y)),R),(X,k)),((H,K),(o,Q))),(((u,a),((9,(S,(1,M))),F)),((I,x),(3,Z)))),((((e,(d,l)),((((s,(z,r)),8),J),4)),((W,f),((c,N),0))),(((i,(j,h)),(T,_)),((((m,(P,6)),q),(y,(E,b))),((G,U),(v,(p,7)))))));
